src/style/components/gauge.module.scss

```diff
@@ -60,6 +67,7 @@
 }
 
 .name {
+    width: 100%;
     position: absolute;
```

Comment:
Are you sure that `position: absolute` is here necessary?

Reply:
I think that yes - name (like PM10) is being put onto canvas so these elements are overlapping. But maybe will it be better to use negative margins with position: static?